src/sagemaker/local/local_session.py

```diff
@@ -63,6 +65,88 @@ def __init__(self, sagemaker_session=None):
         """
         self.sagemaker_session = sagemaker_session or LocalSession()
 
+    def create_processing_job(
+        self,
+        ProcessingJobName,
+        AppSpecification,
+        ProcessingResources,
+        Environment=None,
+        ProcessingInputs=None,
+        ProcessingOutputConfig=None,
```

Comment:
please use pythonic argument names, here and throughout

Reply:
You can't change this here. Reason is that the local sagemaker client gets invoked from:

https://github.com/aws/sagemaker-python-sdk/blob/5b15f3006efe90fbba43da7841ff5f0ad790a78e/src/sagemaker/session.py#L870 

and this is the expected process request:

https://github.com/aws/sagemaker-python-sdk/blob/5b15f3006efe90fbba43da7841ff5f0ad790a78e/src/sagemaker/session.py#L872